internal/app/utils_test.go

```diff
@@ -5,6 +5,71 @@ import (
 	"testing"
 )
 
+func TestReplaceAtIndex(t *testing.T) {
+	type args struct {
+		in string
+		r  rune
+		i  int
+	}
+	tests := []struct {
+		name string
+		args args
+		want string
+	}{
+		{
+			name: "last_colon",
```

Comment:
Yeah, I can actually remove the whole test since this is covered in the one below, it was just that I did this before adding the other method and then because there was coverage there I didn't add any more cases here...

Reply:
I've removed this test since the method is already covered in `TestOciRefToFilename`.